src/Components/NavBar/Menus/__tests__/NewMobileNavMenu.test.tsx

```diff
@@ -56,4 +70,66 @@ describe("MobileNavMenu", () => {
       expect(linkText).not.toContain("Works for you")
     })
   })
+
+  describe("Analytics tracking", () => {
```

Comment:
I'd like to add a few more tests for when a user clicks items from the Artworks or Artists dropdown, but I'm finding  it difficult to setup tests for two or more layers deep. I'm open to suggestions and would love to pair on this.

Reply:
My personal feeling is that testing the analytics calls of these links could go one of two ways: 

* Verify that links are _in general_ hooked up to analytics, which the tests here cover. 
* Verify that _all_ links are comprehensively hooked up to the correct analytics. 

I don't think adding tests of second-level links is necessary unless you're looking to go comprehensive. 

If you _are_ looking to be comprehensive, I think there are some things you could do to these tests to make it easier to write them all:

* Add a helper for finding a specific link by text. This would make it easier to target each individual link/sub-link without being tied to order/sequence of elements.
* Use [`jest.each`](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) to write one test to cover lots of similar test cases. It would save a good amount of copy-paste.

I'd love to pair on it! Feel free to put some time on my calendar.